specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2023-08-01-preview/purviewDataPolicy.json

```
@@ -0,0 +1,616 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview data policies REST API Document",
+    "version": "2023-08-01-preview"
+  },
+  "paths": {
+    "/data-policies": {
+      "get": {
+        "tags": [
+          "List_DataPolicy"
+        ],
+        "summary": "API to list the purview data policies",
+        "description": "List all data policies.",
+        "operationId": "List_DataPolicy",
```

The Azure convention for operationIds is "noun_verb"
```suggestion
        "operationId": "DataPolicy_List",
```

---

updated.